src/Account.java

```diff
@@ -0,0 +1,60 @@
+import java.util.Objects;
+
+class Account implements AccountService {
```

Comment:
what Account class implements AccountService? it's not the past of the task
AccountService might be implemented only by AccountServiceImpl